docs/containers/debug-common.md

```diff
@@ -0,0 +1,118 @@
+---
+Order: 9
+Area: containers
+TOCTitle: Debug
+ContentId: A1371726-5310-4923-B43B-240F36C6264E
+PageTitle: Debug an app running in a Docker container
+DateApproved: 
+MetaDescription: Debug an app running in a Docker container, using Visual Studio Code.
+---
+# Debug apps within Docker containers
+
+With version 0.9.0 and later, the Docker extension provides more support for debugging applications within Docker containers, such as scaffolding `launch.json` configurations for attaching a debugger to applications running within a container.
+
+The Docker extension provides a `docker` debug configuration provider that manages how VS Code will launch an application and/or attach a debugger to the application in a running Docker container. This provider is configured via entries within `launch.json`, with configuration being specific to each application platform supported by the provider.
+
+## Platform Support
+
+The Docker extension currently supports debugging .NET Core and Node.js applications within Docker containers. 
+
+### .NET Core
+
+More information about debugging .NET Core applications within Docker containers can be found [here](/docs/containers/debug-netcore.md).
```

Comment:
Avoid [here] for users with screen readers.
"can be found in [Debug .NET Core within Docker containers].

Reply:
I think there are a couple more instances of [here] in this topic.

Reply:
Thanks, got those now.